OvaryModelGenerator/PelvisModelGenerator.h

```diff
@@ -4,23 +4,24 @@
 #include <iostream>
 
 #include "vtkPolyData.h"
+#include "vtkSmartPointer.h"
 
 class PelvisModelGenerator{
 
-    public:
-        float intercristalDistance;
-        float pelvisScaleFactor;
-        vtkPolyData* pelvisOutputPolydata;
-
     public:
         PelvisModelGenerator();
         ~PelvisModelGenerator();
-        PelvisModelGenerator(const PelvisModelGenerator &other);
+        PelvisModelGenerator(const PelvisModelGenerator &other) = delete;
+        PelvisModelGenerator & operator=(const PelvisModelGenerator &other) = delete;
 
         void SetIntercristalDistancemm(float intercristalDistance); //sets intercristal distance of pelvis and scale factor
         void Generate();
         vtkPolyData* GetOutput();
 
+    private:
+        float m_IntercristalDistance;
+        float m_PelvisScaleFactor;
+        vtkSmartPointer<vtkPolyData> m_PelvisOutputPolydata;
```

Comment:
For data storage, a vtkSmartPointer is recommended rather than just a pointer. This brings convenience in memory management. Without this, an explicit deletion of this pointer should be included in the class' destructor, or there will be memory leak.